components/zip321/CHANGELOG.md

```diff
@@ -32,3 +32,4 @@ The entries below are relative to the `zcash_client_backend` crate as of
   - `zip321::parse::{lead_addr, zcashparam}`
 - `zip321::Param::Memo` now boxes its argument.
 - `zip321::Param::Addr` now wraps a `zcash_address::ZcashAddress`
+- MSRV is now 1.66.0.
```

Comment:
This crate was also recently extracted from another crate with a higher MSRV than other component crates, and currently is synced with the workspace MSRV. It doesn't necessarily need to be though as I expect it to be a bit more stable, but fine to keep it synced for now.